internal/commands/reminder.go

```diff
@@ -215,27 +203,27 @@ func setRecurrence(incident model.Incident) time.Duration {
 }
 
 func sendNotification(ctx context.Context, app *app.App, incident model.Incident) {
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", incident.ChannelId),
+		log.NewValue("channelName", incident.ChannelName),
+		log.NewValue("incident.Status", incident.Status),
```

Comment:
nit: rename to only status to match the pattern

PS: for next steps, another better option would be to actually have a file (like logtags.go) to centralize all the log tags in the form of constants, this way we would ensure consistency...but this is very likely something we don't want to lose time with now during the hackathon :]